### PR TITLE
s/v-sync/Vsync/g

### PR DESCRIPTION
**Pull request by shouichi**
## Description

Only one file uses "v-sync" and others use "Vsync". Fix to use Vsync
sync for consistency.

## Related Issues

None. Just a documentation fix.

## Tests

None. Just a documentation fix.

## Checklist

Before you create this PR confirm that it meets all requirements listed below by checking the relevant checkboxes (`[x]`). This will ensure a smooth and quick review process.

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I signed the [CLA].
- [x] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I updated/added relevant documentation (doc comments with `///`).
- [ ] All existing and new tests are passing.
- [ ] The analyzer (`flutter analyze --flutter-repo`) does not report any problems on my PR.
- [x] I am willing to follow-up on review comments in a timely manner.

## Breaking Change

Did any tests fail when you ran them? Please read [Handling breaking changes].

- [x] No, no existing tests failed, so this is *not* a breaking change.
- [ ] Yes, this is a breaking change. *If not, delete the remainder of this section.*
   - [ ] I wrote a design doc: https://flutter.dev/go/template *Replace this with a link to your design doc's short link*
   - [ ] I got input from the developer relations team, specifically from: *Replace with the names of who gave advice*
   - [ ] I wrote a migration guide: *Replace with a link to your migration guide*

[issue database]: https://github.com/flutter/flutter/issues
[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Test Coverage]: https://github.com/flutter/flutter/wiki/Test-coverage-for-package%3Aflutter
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Handling breaking changes]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes

### CODE REVIEW

File: packages/flutter_test/lib/src/widget_tester.dart [hunk 1]
```diff
@@ -327,7 +327,7 @@ class WidgetTester extends WidgetController implements HitTestDispatcher, Ticker
   /// Triggers a frame after `duration` amount of time.
   ///
   /// This makes the framework act as if the application had janked (missed
-  /// frames) for `duration` amount of time, and then received a v-sync signal
+  /// frames) for `duration` amount of time, and then received a Vsync signal
```
Comment (goderbauer) [inline packages/flutter_test/lib/src/widget_tester.dart]: nit:

```suggestion
  /// frames) for `duration` amount of time, and then received a vsync signal
```

Comment (shouichi) [reply packages/flutter_test/lib/src/widget_tester.dart]: In the documentation, capital "Vsync" is used and I followed that. Should it be lower "vsync"?

https://github.com/flutter/flutter/blob/01953a101576a64aeaec48d434e780ce31e50538/packages/flutter/lib/src/scheduler/binding.dart#L764

**Review by goderbauer — APPROVED** [review]
LGTM

Looks like we use both versions in the docs.

**Comment by Piinks** [comment]
Hi @shouichi Thanks for the contribution! 

It looks like your branch contains an unrelated bug that is causing the tests here to fail. It has been patched though. If you update your branch with the latest changes on master, it should be resolved. 

Let me know if you need any help with this. :)

**Comment by shouichi** [comment]
@Piinks Rebased to the current master and force pushed. Thanks!

**Pull request merged**